frontend/src/components/CollabPage/CollabPage.jsx

```diff
@@ -39,8 +41,14 @@ const connectSessionSocket = () => {
 };
 
 export default function CollabPage() {
+  const { user } = useAuthContext();
+  const username = user == null ? null : user.username;
+  let questionId = null;
```

Comment:
FYI not sure if you realise we're using redux, but you can retrieve the questionId using a selector as seen in `QuestionPanel.jsx`. Unless there was a conscious decision on your part to do so, I'll change it myself

```
const questionId = useSelector(selectQuestionId);
```